### DISCUSSION TITLE
ScriptReference/MonoBehaviour.OnBecameVisible

### DISCUSSION THREAD

**Discussion by giscus** [Developer-Notes-Extension/Unity-Notes#528] [Notes]
# ScriptReference/MonoBehaviour.OnBecameVisible

https://docs.unity3d.com/ScriptReference/MonoBehaviour.OnBecameVisible.html

**Comment by abrusle** [comment]
From my testing:
- Doesn't seem to work with CanvasRenderers
- Is not triggered when the gameObject goes out from or comes into the camera's view
- Doesn't account for SpriteMasks
